src/Libs/GObject-2.0/Public/Type.cs

```diff
@@ -51,9 +52,9 @@ public Type(nuint value)
     #endregion
 
     // Print out the name of the GType
-    public override string ToString()
+    public override string? ToString()
     {
-        return StringHelper.ToStringUtf8(Internal.Functions.TypeName(_value));
+        return Internal.Functions.TypeName(_value).ConvertToString();
```

Comment:
The Typename can't be null I think? With the current code it would be correct to throw an exeption in case of a null.

I think we need some new classes to improve the handling of `null` strings:
- `NullablePlatformStringOwnedHandle` and `NonNullablePlatformStringOwnedHandle` 
-  `NullableUtf8StringOwnedHandle` and `NonNullableUtf8StringOwnedHandle`

Those classes could either return `null` or throw an exception in the `ConvertToString` method. The classes would need to be generated according to the nullable annotation.